frontend/ui/data/css_tweaks.lua

```diff
@@ -10,10 +10,11 @@ They may have the following optional attributes:
 ]]
 
 local _ = require("gettext")
+local C_ = _.pgettext
 
 local CssTweaks = {
     {
-        title = _("Pages"),
+        title = C_("Menu|StyleTweaks|", "Pages"),
```

Comment:
Why not just `C_("StyleTweaks", "Pages")` ?
The | separators  just add room for errors.

And in that context, we define a list of tweak categories and tweaks, which happen to be used as menu items, but I'd rather not see "Menu" mentionned.
In the case there would be a tweak also named "Pages", I'd rather see `C_("StyleTweaksCategory", "Pages")`

Reply:
It could also just be `Style tweaks category` or `style tweaks category`. We just want to try to avoid descriptive sentences that would be more likely to (want to be) change.